packages/flutter_tools/lib/src/devfs.dart

```diff
@@ -16,6 +17,13 @@ import 'vmservice.dart';
 
 typedef void DevFSProgressReporter(int progress, int max);
 
+class DevFSConfig {
+  /// Should DevFS assume that symlink targets are stable?
+  bool cacheSymlinks = false;
+}
+
+DevFSConfig get devFSConfig => context[DevFSConfig];
```

Comment:
I kind of want globals.dart to die so I wanted to keep it local to where it is actually used.